lib/test-rule.js

```diff
@@ -32,3 +45,20 @@ module.exports = function testRule({ moddleRoot, rule }) {
 
   return reporter.messages;
 };
+
+
+// helpers ///////////////////////////
+
+function stringifyPath(path) {
```

Comment:
We don't need this function but if we did it could be simplified to https://github.com/bpmn-io/bpmn-js-properties-panel/blob/spike-events/src/utils/PathUtil.js#L84.